src/main/java/org/polyfrost/hytils/util/WaypointUtil.java

```diff
@@ -278,22 +352,24 @@ public static void renderWayPoint(String str, Vector3f loc, float partialTicks)
         GlStateManager.translate(x, y, z);
         GlStateManager.translate(0, viewer.getEyeHeight(), 0);
 
-        renderNametag(str);
+        OneColor color = new OneColor(-1);
```

Comment:
put this onecolor instance somewhere else so that its not being created every frame this is being drawn